torchaudio/prototype/models/_emformer_hubert.py

```diff
@@ -0,0 +1,323 @@
+from typing import List, Optional, Tuple
+
+import torch
+from torchaudio.models import Wav2Vec2Model
+from torchaudio.models.emformer import Emformer
+from torchaudio.models.rnnt import _TimeReduction
+
+
+class FeatureEncoder(torch.nn.Module):
+    """Extract features from log mel-spectrogram input. Consists of linear layer and time reduction layer.
```

Comment:
nit: I normally see `logmel spectrogram` or `log-mel spectrogram` and not `log mel-spectrogram`?

Reply:
that's right, will change it to log-mel spectrogram.